README.md

```diff
@@ -121,6 +122,21 @@ _note: if .dropOffDate is not set, it will be calculated with .pickUpDate + .dur
 |         | .dateFrom    | '20092017' 
 |         | .dateTo      | '20092017' 
 
+
+| type      | fields       | example
+| :---      | :---         | :---
+|'roundTrip'| .bookingId   | 'NEZE2784NQXTHEN' 
+|           | .destination | 'YYZ' 
+|           | .no          | '1' 
+|           | .startDate   | '05122017' 
+|           | .endDate     | '16122017'
+|           | .title       | 'H'
```

Comment:
the following fields are not part of a service. they are parts of the "customer data". we should separate them from the service data.